### DISCUSSION TITLE
Minimum Data about a Digital Specimen (MIDS) Evaluation Application

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4950] [Announcements]
This is a cool tool for evaluating records at GBIF and their level of "completion".

[YouTube Demo](https://t.co/x7ArT4RDC8)

[Github Repo](https://github.com/AgentschapPlantentuinMeise/MIDSCalculator)

**Comment by ewommack** [comment]
Hmmm, nifty. Potential Newsletter article material?

**Reply by Jegelewicz** [reply]
POssibly? But also - could we integrate this right into Arcots?
